VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.html

```diff
@@ -1,13 +1,46 @@
 <div class="container">
     <h2 class="govuk-heading-s white">{{ 'private-consultation-participants.participants' | translate }}</h2>
-    <div class="participant-grid">
-        <ng-container *ngFor="let participant of getPrivateConsultationParticipants(); trackBy: trackParticipant">
+    <div class="participant-grid">        
+        <ng-container *ngFor="let groupMember of getGroupMembers();">
+            <ng-container *ngIf="isJohConsultation() && geMemberConsultationParticipants(groupMember).length > 0">
+                <div  class="language-icon">
+                    <img  class="small-crest" src="/assets/images/UkGovCrestWhite.png"[attr.alt]="'private-consultation-participants.crest-icon-alt' | translate" />
```

Comment:
Is it correct that this image is only duplicated per group member, instead of per participant?

Reply:
fixed